data/Britain1/Britain1.html

```diff
@@ -32,89 +30,62 @@ <h2>Britain</h2>
   <table>
     <tr>
       <td>
-        <a href="../Britain_Complex/Britain_Complex.html"
-          >Complex<br/><img
-            style="width: 45%"
-            src="../Britain_Complex/Content/Images/daf_1.jpeg"
-        /></a>
+        <a href="../Britain_Cmplx/Britain_Cmplx.html">Complex<br /><img style="width: 45%"
+            src="../Britain_Cmplx/Content/Images/daf_1.jpeg" /></a>
       </td>
       <td>
-        <a href="../Britain_Standalones/Britain_Standalones.html"
-          >Standalones<br/><img
-            style="width: 45%"
-            src="../Britain_Standalones/Content/Images/oscill_c.jpg"
-        /></a>
+        <a href="../Britain_Standalones/Britain_Standalones.html">Standalones<br /><img style="width: 45%"
+            src="../Britain_Standalones/Content/Images/oscill_c.jpg" /></a>
       </td>
     </tr>
     <tr>
       <td>
-        <a href="../Britain_Composite/Britain_Composites.html"
-          ><img
-            style="width: 45%"
-            src="../Britain_Composite/Content/Images/unit_dar.jpeg"
-          /><br/>Composites</a
-        >
+        <a href="../Britain_Composite/Britain_Composites.html"><img style="width: 45%"
+            src="../Britain_Composite/Content/Images/unit_dar.jpeg" /><br />Composites</a>
       </td>
       <td>
-        <a href="../Britain_Legacy/Britain_Legacy.html"
-          ><img
-            style="width: 45%"
-            src="../Britain_Legacy/Content/Images/unit_da.jpg"
-          /><br/>Legacy</a
-        >
+        <a href="../Britain_Legacy/Britain_Legacy.html"><img style="width: 45%"
+            src="../Britain_Legacy/Content/Images/unit_da.jpg" /><br />Legacy</a>
       </td>
     </tr>
   </table>
 </div>
 
 This page has a special layout - since it starts by splitting the plants/units
-into their separate types, then provides a dedicated page for each sub-type. 
+into their separate types, then provides a dedicated page for each sub-type.
 <p>
-Above this text is a hidden table that collates the necessary tables and links.
+  Above this text is a hidden table that collates the necessary tables and links.
 </p>
 <!-- start off with some irrelevant links -->
 <!-- 1. in child of current folder -->
 <div>
-  <a href="./Content/Images/Flag.jpg"
-    ><img src="../Britain_Complex/Content/Images/unit_da.jpg" />
+  <a href="./Content/Images/Flag.jpg"><img src="../Britain_Cmplx/Content/Images/unit_da.jpg" />
   </a>
 </div>
 <!-- missing image child -->
-<div><a href="../Britain_Complex/Britain_Complex.html">Here</a></div>
```

Comment:
Hello @AbegaM - please note that we cannot rely on the page name matching the folder name below.  So, could you please make edits so that the page and folder are different?  Maybe the page should be `Britain_Complx1.html`.   You'll have to change the `href` below and the filename ;-)

Reply:
@IanMayo Let me know if there is an issue with the new update i pushed